bundles/mapping/mapmodule/plugin/layers/BackgroundLayerSelection.jsx

```diff
@@ -48,14 +49,15 @@ export const BackgroundLayerSelection = ({ isMobile = false, layers, current, ma
                         visible={true}
                         onClick={() => layer.onClick(layer.id)}
                         icon={null}
-                        text={layer.title}
                         active={Number.parseInt(layer.id, 10) === current?.getId()}
                         loading={false}
                         $isDropdown={false}
                         $minWidth={BUTTON_WIDTH}
                         data-layerid={layer.id}
                         {...rest}
-                    />
+                    >
+                        {layer.title}
```

Comment:
Do we need the ellipsis handling added here?

Reply:
added